com.unity.multiplayer.mlapi/Runtime/Spawning/SpawnManager.cs

```diff
@@ -323,81 +323,88 @@ internal static NetworkedObject CreateLocalNetworkedObject(bool softCreate, ulon
         // Ran on both server and client
         internal static void SpawnNetworkedObjectLocally(NetworkedObject netObject, ulong networkId, bool sceneObject, bool playerObject, ulong? ownerClientId, Stream dataStream, bool readPayload, int payloadLength, bool readNetworkedVar, bool destroyWithScene)
         {
-            if (netObject == null)
+            try
```

Comment:
ugh, I didn't like try-catch approach here at all. why we even throw exceptions below? :(

Reply:
Removed.